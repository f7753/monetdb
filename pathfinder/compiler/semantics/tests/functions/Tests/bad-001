#!/bin/bash

pf -s6 <<'EOT'
declare function foo ($x as xs:string) as xs:string
{
    "foo"
};
declare function foo ($x as xs:integer) as xs:string
{
    "foo"
};
foo (42), foo ("bar")
EOT
echo $?
