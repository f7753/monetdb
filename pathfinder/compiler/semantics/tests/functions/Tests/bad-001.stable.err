stderr of test 'bad-001` in directory 'compiler/semantics/tests/functions` itself:


# 16:19:34 >  
# 16:19:34 >  Mtimeout -timeout 60 ./bad-001 bad-001 
# 16:19:34 >  

function redefined: `fn:foo'
# halted in functions.c (fun_add_user), line 458

# 16:19:34 >  
# 16:19:34 >  Done.
# 16:19:34 >  

