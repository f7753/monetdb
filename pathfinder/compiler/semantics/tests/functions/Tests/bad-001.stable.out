stdout of test 'bad-001` in directory 'compiler/semantics/tests/functions` itself:


# 16:19:34 >  
# 16:19:34 >  Mtimeout -timeout 60 ./bad-001 bad-001 
# 16:19:34 >  

15

# 16:19:34 >  
# 16:19:34 >  Done.
# 16:19:34 >  

